cypress/e2e/Authentication/login.cy.ts

```diff
@@ -1,3 +1,26 @@
+// TODO: How to reuse the component's test ids
```

Comment:
I'll put an improvement in another PR 🙏

Reply:
After adding it, it does mess with the code coverage 🥲 I'm gonna close that Pr and maybe put some summarize of the issue here 📝